sparkmagic/setup.py

```diff
@@ -85,6 +85,7 @@ def version(path):
           'pandas>=0.17.1',
           'numpy',		
           'requests',
+          'requests-kerberos',
```

Comment:
You will need to bump the version of all 3 packages in this code base from `0.10.0` to `0.11.0`